app/src/main/res/values-night/colors.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <color name="colorPrimary">#212121</color>
-    <color name="colorAccent">#eeeeee</color>
+    <color name="colorAccent">#899ab4</color>
```

Comment:
![screenshot_2017-03-01-03-33-49-177_com zulip android dev](https://cloud.githubusercontent.com/assets/17759582/23585034/b2fd160c-0198-11e7-8e18-bd13aa7a389f.png)
![screenshot_2017-03-01-02-54-12-546_com zulip android dev](https://cloud.githubusercontent.com/assets/17759582/23585035/b2fed726-0198-11e7-820e-7b454fa9498a.png)

I changed accent colour because it was very hard to see links in previous accent colour.